Exiled.Events/Patches/Generic/PlayerHasHint.cs

```diff
@@ -30,7 +30,7 @@ internal static class PlayerHasHint
         private static void Postfix(HintDisplay __instance, Hint hint)
         {
             // Try to get the player, if it doesn't exist, just return
-            if (__instance.gameObject is null || !(Player.Get(__instance.gameObject) is Player player))
+            if (__instance == null || !(Player.Get(__instance.gameObject) is Player player))
```

Comment:
is null

Reply:
__instance?.gameObject is null
would be better.

Reply:
okay, will change that quick